packages/mui-base/src/useList/useListItem.types.ts

```diff
@@ -45,8 +39,4 @@ export interface UseListItemReturnValue {
    * If `true`, the current item is selected.
    */
   selected: boolean;
-  /**
-   * The ref to the root element.
-   */
-  rootRef: React.RefCallback<Element> | null;
```

Comment:
Why were the refs removed? Were they used for the bus communication system? Or were they not required before either?

Reply:
It wasn't used at all. It must have been a leftover from a previous implementation.